src/components/data/SpeakerData.js

```diff
@@ -1,113 +1,93 @@
 const SpeakerData = [
     {
         id: 1,
-        name: "Nate Bolt",
+        name: "Dr. David Danks",
         day: 1,
-        event: "Day 1 (4/26) Keynote Speaker",
-        img: "NateBolt.jpg",
-        title: "Founder, Ethnio",
+        event: "Day 1 (5/1) Keynote Speaker",
+        img: "DavidDanks.jpg",
+        title: "Professor of Data Science and Philosophy @UCSD",
         bio: "",
-        link: "https://www.linkedin.com/in/boltron/"
+        link: "https://www.linkedin.com/in/david-danks-21538613"
     },
     {
         id: 2,
-        name: "Dr. Ben Cipollini",
+        name: "Dr. Linda Abarbanell",
         day: 1,
-        event: "Day 1 Workshop: Transitioning from Academia to Industry",
-        img: "BenCipollini.jpg",
-        title: "Data Science Manager at Facebook",
+        event: "Day 1 Workshop: Language, Culture, and Cognition Talk",
+        img: "LindaAbarbanell.jpg",
+        title: "Professor of Psychology @SDSU",
         bio: "",
-        link: "https://www.linkedin.com/in/social-sector-data-insights/"
+        link: ""
     },
     {
         id: 3,
-        name: "Vasudev Lal",
+        name: "Meghan Yiu",
         day: 2,
-        event: "Day 2 (4/27) Keynote Speaker",
-        img: "VasudevLal.jpg",
-        title: "AI/ML Research Scientist at Intel Labs",
+        event: "Day 2 (5/2) Workshop: UX Pathways Talk",
+        img: "MeghanYiu.jpeg",
+        title: "User Experience Designer @Epic",
         bio: "",
-        link: "https://www.linkedin.com/in/vasudev-lal-79bb336/"
+        link: "https://www.linkedin.com/in/meghanyiu/"
     },
     {
         id: 4,
-        name: "Stephanie Alfonso",
+        name: "Kayla Monnette",
         day: 2,
-        event: "Day 2 Workshop: Neuroscience",
-        img: "StephanieAlfonso.jpg",
-        title: "Instructor at UCSD",
+        event: "Day 2 Workshop: UX Pathways Talk",
+        img: "KaylaMonnette.png",
+        title: "User Experience Designer @Epic",
         bio: "",
-        link: "https://www.linkedin.com/in/stephanie-alfonso-19793bab/"
+        link: "https://www.linkedin.com/in/kaylamonnette/"
     },
     {
         id: 5,
-        name: "Ana Chkhaidze",
-        day: 3,
-        event: "Career Panel",
-        img: "AnaChkhaidze.jpg",
-        title: "PhD Student at UCSD",
+        name: "Sam Hande",
+        day: 2,
+        event: "TBD",
```

Comment:
I think we can just replace TBD with Day 2 workshop or something - might sound better than TBD?